docs/DeveloperGuide.adoc

```diff
@@ -295,53 +296,74 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |Computing student |plan for specialisations |complete the requirements for my specialisation
 
-|`* * *` |user |add a new person |
+|`* * *` |student |see the overview of my degree progression |keep in mind my progress in school
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |student |choose modules to take for future semesters |plan for future semesters
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |student |store my past grades |calculate my overall <<CAP, CAP>> every semester
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |student |view <<prerequisite, prerequisites>> of every module |know what modules I should complete early
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+|`* * *` |student |view <<preclusion, preclusions>> of every module |know what modules I should not take
+
+|`* *` |student |maintain a list of unfinished homework and their deadlines |submit my assignments in time
 
-_{More to be added}_
+|`* *` |student |pool notes for my modules together |easily find my notes for each module
+
+|`*` |student |see the modules that I have taken |
```

Comment:
This is currently missing the "So that I can…​" part, which could be: "know what modules I have completed or been exempted from".

Reply:
This could also be a slightly higher priority Use Case, as welll.